src/test_utils.rs

```diff
@@ -375,8 +392,9 @@ impl TestContext {
     // This code is mainly the same as `log_msglist` in `cmdline.rs`, so one day, we could
     // merge them to a public function in the `deltachat` crate.
     #[allow(dead_code)]
-    pub async fn print_chat(&self, chat: &Chat) {
-        let msglist = chat::get_chat_msgs(self, chat.get_id(), 0x1, None).await;
+    #[allow(clippy::clippy::indexing_slicing)]
+    pub async fn print_chat(&self, chat_id: ChatId) {
```

Comment:
not sure if that is needed for tests, it is okay for debugging in repl tool, but i would remove that from the tests.

Reply:
It was also very useful for debugging tests :)

Or do you mean, remove the function call from the tests? In this case, of course, I just forgot to remove them.

Reply:
i meant both :) but if the function call is removed, the remaining dead test-code is maybe okay, if it is useful for development at least one devs :)

Reply:
Yup, it's useful for development for at least one dev :wink: